### PR TITLE
Adds ported clothes from Polaris

### PR DESCRIPTION
**Pull request by BoryaTheSlayer**
Adds four pieces of clothing, ported from Polaris: blue dress, dark red dress, crop top, crop dress.

### CODE REVIEW

**Comment by BoryaTheSlayer** [comment]
!review

**Comment by BoryaTheSlayer** [comment]
> crop tops are haram

I AM willing to remove the crop tops, as I wasn't sure about adding them myself.

**Comment by alsoandanswer** [comment]
arent things like these already in-game? why are we adding even more

**Comment by BoryaTheSlayer** [comment]
> arent things like these already in-game? why are we adding even more
> 
> they don't look very good, IMHO

No, actually! We do not have anything like this in game. Our current dresses, imo, are even worse.
I wouldn't mind help from spriters to make this better 👀

**Comment by OneOneThreeEight** [comment]
> > crop tops are haram
> 
> I AM willing to remove the crop top, as I wasn't sure about adding them myself. The crop dress stays tho. It good.

I'm joking, I don't care if it stays or not. Though I still remember the guy that got banned whose shitty waifu wore a "skimpy dress" and that was hilarious.

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -775,4 +775,46 @@
...
+
+
+/obj/item/clothing/under/croptop
+	name = "crop top"
+	desc = "Light shirt which shows the midsection of the wearer."
+	icon_state = "croptop"
+	item_state = "croptop_s"
+	worn_state = "croptop"
+
+
```
Comment (NonQueueingMatt) [inline code/modules/clothing/under/miscellaneous.dm]:
```suggestion
```

**Pull request merged**